src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -220,6 +228,26 @@ module sonic-acl {
 				leaf INNER_L4_DST_PORT {
 					type uint16;
 				}
+
+				leaf VLAN_ID {
```

Comment:
>VLAN_ID [](http://example.com/codeflow?start=9&length=7)

This change is not related to your PR title or description. 
Could you focus on MAC part as original commit?

Reply:
VLAN ID is part of the MAC header. This PR adds various other MAC header keys like PCP DEI etc. I will update the description to explicitly list all added fields

Reply:
Also update the PR title?

Reply:
Done